### PR TITLE
Change File->Preferences->Settings to File->Settings

### PR DESCRIPTION
**Pull request by wyattscarpenter**
I should note that I have not tried to compile this code, I am hoping the change is simple enough to not be wrong.

### CODE REVIEW

File: IDE/src/IDEApp.bf [hunk 1]
```diff
@@ -5006,9 +5006,7 @@ namespace IDE
             AddMenuItem(subMenu, "&Save File","Save File", new => UpdateMenuItem_HasActiveDocument);
             AddMenuItem(subMenu, "Save &As...", "Save As", new => UpdateMenuItem_HasActiveDocument);
 			AddMenuItem(subMenu, "Save A&ll", "Save All");
-			let prefMenu = subMenu.AddMenuItem("&Preferences");
-			//prefMenu.AddMenuItem("&Keyboard Shortcuts", null, new (evt) => { ShowKeyboardShortcuts(); });
-			AddMenuItem(prefMenu, "&Settings", "Settings");
+			AddMenuItem(subMenu, "S&ettings", "Settings");
```
Comment (JamesOrson) [inline IDE/src/IDEApp.bf:5009]: It was `"&Settings"` before. Is `"S&ettings"` intentional?

Comment (wyattscarpenter) [reply IDE/src/IDEApp.bf:5009]: Yes, otherwise it would conflict with the s in save, right?

Comment (JamesOrson) [reply IDE/src/IDEApp.bf:5009]: I am not sure how this stuff works. Was more curious as to the reasoning why the & was moved.

Comment (wyattscarpenter) [reply IDE/src/IDEApp.bf:5009]: I am also not sure how this works, but the impression I got was that the & controls the letter that gets to be the hotkey, so I had to make "&Settings" into "S&ettings" so it wouldn't conflict with "&Save File" now that they are on the same level of the menu.

**Comment by wyattscarpenter** [comment]
There are more items in the prefMenu now so this change no longer makes sense.

**Pull request closed without merging**